tfjs-backend-wasm/README.md

```diff
@@ -108,13 +108,13 @@ JS backend, and ~5.3-7.7X slower than the WebGL backend.
 
 <img src="./mobilenet-v2-bench.svg">
 
-| MobileNet inference (ms) | WASM  | WebGL | Plain JS |
-|--------------------------|-------|-------|----------|
-| iPhone X                 | 147.1 | 20.3  | 941.3    |
-| iPhone XS                | 140   | 18.1  | 426.4    |
-| Pixel 3                  | 266.2 | 77.3  | 2345.2   |
-| Desktop Linux            | 91.5  | 17.1  | 1049     |
-| Macbook Pro              | 98.4  | 19.6  | 893.5    |
+| MobileNet inference (ms) | WASM  | WebGL | Plain JS | WASM + SIMD |
+|--------------------------|-------|-------|----------|-------------|
+| iPhone X                 | 147.1 | 20.3  | 941.3    | N/A         |
+| iPhone XS                | 140   | 18.1  | 426.4    | N/A         |
+| Pixel 3                  | 266.2 | 77.3  | 2345.2   | N/A         |
```

Comment:
Chrome on Android supports SIMD

Reply:
Hmm, for some reason the support test I'm using (https://github.com/GoogleChromeLabs/wasm-feature-detect) returns false on Chrome Android, even though in chrome://flags it says it's supported.

Reply:
Ah turns out the feature test only works in Chrome 84+, and I was using Chrome 80. Will work on getting these benchmarks :)